pages/tools/cli.md

```diff
@@ -582,65 +582,82 @@ continuously stream output
 
 # Sync
 
-## sync [source]
+## sync [uuid]
 
 WARNING: If you're running Windows, this command only supports `cmd.exe`.
 
 Use this command to sync your local changes to a certain device on the fly.
 
-The `source` argument can be either a device uuid or an application name.
+After every 'resin sync' the updated settings will be saved in
+'<source>/.resin-sync.yml' and will be used in later invocations. You can
```

Comment:
right, we changed it to a dotfile to follow the convention of having dotfiles for third-party app config files (e.g. `.gitignore`, `.*rc files etc) and it doesn't parse `resin-sync.yml` files - only `.resin-sync.yml`